roles/satellite-clone/tasks/backup_check.yml

```diff
@@ -97,4 +91,4 @@
       Standard backup scenario - config_files.tar.gz, mongo_data.tar.gz, pgsql_data.tar.gz, (optional) pulp_data.tar
       Online backup scenario - config_files.tar.gz, mongo_dump folder, foreman.dump, candlepin.dump, (optional) pulp_data.tar
       RHEL 6 to 7 migration scenario - config_files.tar.gz, mongo_data.tar.gz, foreman.dump, candlepin.dump, (optional) pulp_data.tar
```

Comment:
mongo_data.tar.gz should be changed to folder here and this line can be merged with the previous one.